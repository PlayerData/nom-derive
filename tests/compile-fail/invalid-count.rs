extern crate nom;
extern crate nom_derive;
use nom_derive::Nom;

pub struct Toto;

#[derive(Nom)]
pub struct S {
    #[nom(Count = "4")]
    a: u32,
}

fn main() {}
