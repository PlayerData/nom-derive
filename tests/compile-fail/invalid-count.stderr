error: Unable to infer parser for 'Count' attribute. Is item type a Vec ?
  --> $DIR/invalid-count.rs:10:8
   |
10 |     a: u32,
   |        ^^^
